app/ui/src/main/res/layout/message_list.xml

```diff
@@ -1,29 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
+<RelativeLayout
```

Comment:
What's the reason for switching this to `RelativeLayout`?

Reply:
I used it so I can align the progress bar in front of the other Views (z direction). Just removing maxHeight produced ugly white space above the list.

Reply:
Oh, I see. The progress bar is now covering part of the toolbar and part of the list.

Reply:
The visual part of the progress bar is only covering the list because its background is semi transparent. I tried moving it up a bit to cover the toolbar, too, but that looked strange because the progress bar then had a visible line in the middle. The toolbar is covered a bit by spacing around the progress bar that can not be disabled (you can work around that by setting the height to something smaller than wrap_content like before but I do not think that this is clean). Basically, this PR makes the progress bar a bit bigger but without causing additional whitespace on top of the list.

Reply:
It looks good :+1: